Describe Satellite usage metrics collection (#3846)
theforeman/foreman-documentation: guides/common/modules/ref_usage-metrics-collected-by-project.adoc

* `assigned_custom_roles_count`
* `audits|max_created_at`
* `audits|min_created_at`
* `audits|records_count`
* `bookmarks_by_public_by_type|public|User`
* `compliance_arf_report_last_year_count`
* `compliance_non_default_scap_contents_count`
* `compliance_policy_count`
* `compliance_policy_with_tailoring_file_count`
* `compliance_scap_contents_count`
* `compute_resource_vmware_count`
* `custom_roles_count`
* `discovery_rules_count`
* `external_user_group_mapping_count`
* `facts_by_type|Puppet|max_update_time`
* `facts_by_type|Puppet|min_update_time`
* `facts_by_type|Puppet|values_count`
* `foreman_virt_who_configure_ahv_count`
* `foreman_virt_who_configure_configurations_count`
* `foreman_virt_who_configure_esx_count`
* `foreman_virt_who_configure_fakevirt_count`
* `foreman_virt_who_configure_hyperv_count`
* `foreman_virt_who_configure_kubevirt_count`
* `foreman_virt_who_configure_libvirtd_count`
* `host_collections_count`
* `host_collections_with_limit_count`
* `hostgroup_max_nesting_level`
* `hostgroup_nesting`
* `hosts_by_compute_profile|none`
* `hosts_by_compute_resources_type|baremetal`
* `hosts_by_managed_count|unmanaged`
* `hosts_by_os_count|RedHat`

> * `hosts_by_type_count|Managed`

Author: Added

[id="usage-metrics-collected-by-{project-context}"]
= Usage metrics collected by {Project}

The `satellite-maintain report generate` command performs the following operations to collect usage metrics:

* `Check how resources are grouped`
* `Check if OpenSCAP is used`
* `Check if recurring logics are used`
* `Check if virt-who is being used and what hypervisor types are present`
* `Check if vmware compute resource is used`
* `Checks the RBAC use`
* `Checks the use of External auth source`
* `Checks the use of Kerberos`
* `Checks the use of Keycloak/OIDC`
* `Checks the use of LDAP auth sources`
* `Facts about hosts and the rest of the inventory`
* `Provisioning facts about the system`
* `Report about platform usages`
* `Report information about the instance itself`

These operations collect the following set of data.

> Items whose key includes vertical bar (`|`) are dynamic values generated based on the contents of the database in your environment:

Reviewer: ```suggestion
Items that include a vertical bar (`|`) are dynamic values generated based on the contents of the database in your environment:
```
